src/asmjs/asm_v_wasm.cpp

```diff
@@ -80,7 +80,7 @@ char getSig(Type type) {
       return 'V';
     case Type::funcref:
       return 'F';
-    case Type::anyref:
+    case Type::externref:
       return 'A';
```

Comment:
`A` here was for `anyref`, but now it is `externref`... but `E` is taken with `exnref`. How about `T`? (This does not affect the output program, so this is just our internal thing)

Reply:
Good eye. X seems good.